result/lldpneighbors.yaml

```diff
@@ -184,6 +184,11 @@ components:
             The organizationally defined subtype field shall contain a unique subtype value assigned by the defining organization.
           type: string
           x-field-uid: 3
+        information:
+          description: >-
+            Contains either the Alpha-numeric information encoded in UTF-8 (as specified in IETF RFC 3629) or include one or more information fields with their associated field-type identifiers designators.
```

Comment:
This contains the remaining bytes of the received Org. Spec.. TLV after the sub-type. The value should be returned in hex format in lower case.